TombEngine/Objects/TR3/Entity/tr3_shiva.cpp

```diff
@@ -386,17 +411,16 @@ namespace TEN::Entities::Creatures::TR3
 				if (item->HitStatus)
 					creature->Flags = 4;
 
-				if (AI.bite && AI.distance < pow(SECTOR(1.25f), 2) ||
-					(item->Animation.FrameNumber == g_Level.Anims[item->Animation.AnimNumber].frameBase &&
-						!creature->Flags))
+				if ((AI.bite && AI.distance < SHIVA_DOWNWARD_ATTACK_RANGE) ||
```

Comment:
It seems that `SHIVA_GRAB_ATTACK_RANGE` should be used here instead of `SHIVA_DOWNWARD_ATTACK_RANGE`.

Reply:
Fixed